components/dashboard/sections/transactions/TransactionsTable.tsx

```diff
@@ -299,14 +299,17 @@ export default function TransactionsTable({
         data={transactions?.nodes || []}
         loading={loading}
         nbPlaceholders={nbPlaceholders}
-        onClickRow={(row, menuRef) => openDrawer(row.id, menuRef)}
+        onClickRow={(row, menuRef) => {
+          openDrawer(row.id, menuRef);
+        }}
         onHoverRow={row => setHoveredGroup(row?.original?.group ?? null)}
```

Comment:
I caught something while testing the performance of the Transactions dashboard: `onHoverRow` causes the whole table to re-render when hovering over a different group. This is fine for now because we do not display a lot of transactions in the table right now, but it could become an issue if we decide to increase the number of rows being displayed at the same time. A possible solution would be implementing this effect using CSS selectors and data properties.

Reply:
Good point, created an issue to track this: https://github.com/opencollective/opencollective/issues/7405